.travis.yml

```diff
@@ -44,7 +44,22 @@ matrix:
         - SPLIT="2/2" TEST_SYMPY="true"
 
     - stage: test
-      python: 2.7
+      python: 3.8-dev
+      dist: xenial
+      sudo: true
+      env:
+        - TEST_DOCTESTS="true" FASTCACHE="false" TEST_SETUP="true"
+    - python: 3.8-dev
+      dist: xenial
+      sudo: true
+      env:
+        - SPLIT="2/2" TEST_SYMPY="true"
```

Comment:
1/2

Reply:
Good catch. I don't know how that typo happened.